### ISSUE TITLE
Invoice DueDate format error

### ISSUE THREAD

**Issue by sminnee** [singer-io/tap-xero#68]
This integration running on StitchData has started throwing the following error::

```
2020-10-26 20:00:38,522Z    tap - INFO HTTP request to "invoices" endpoint took 1.704s, returned status code 200
2020-10-26 20:00:38,673Z    tap - WARNING Unknown string format: /Date(-80006400000+0000)/, (/Date(-80006400000+0000)/)
2020-10-26 20:00:38,676Z    tap - CRITICAL Errors during transform
2020-10-26 20:00:38,677Z    tap - 	DueDate: data does not match {'selected': True, 'type': ['string', 'null'], 'inclusion': 'available', 'format': 'date-time'}
```

It looks like Xero has changed the output format of this field.

**Comment by sminnee** [comment]
Note that excluding Invoice.DueDate from the imported fields avoided this error, which has been my short-term fix.

**Comment by sminnee** [comment]
I suspect that the change here needs to be applied to invoice due-date too 59484e0652acac2b8d3f683ef2a6e62ddb005688

**Comment by sminnee** [comment]
On the face of it it looks like the regex here:
https://github.com/singer-io/tap-xero/blob/master/tap_xero/client.py#L19

Should be changed from:
` r'Date\((\d+)([-+])?(\d+)?\)'`

to:
` r'Date\((-?\d+)([-+])?(\d+)?\)'`

**Comment by sminnee** [comment]
I believe the date represented is `6/20/1967, 12:00:00 PM`, which is likely to be a data error on our part but is a legal date.

**Comment by briansloane** [comment]
Your suggested change sounds like it would work. Can you submit a PR with logs showing that it worked and a unit test for the scenario, confirming it returns the correct datetime value?

**Comment by sminnee** [comment]
Will do. I normally use it via StitchData; will you be pushing a 2.0.1 soon or did you want me to test first?

**Comment by dmosorast** [comment]
Cleaning up while I'm here, this was addressed in #74

**Comment by sminnee** [comment]
Great, thank you!
